src/chart/generateCategoricalChart.tsx

```diff
@@ -2184,6 +2185,18 @@ export const generateCategoricalChart = ({
             }),
           ];
         }
+      } else if (hasActive) {
+        /**
+         * We hit this block if consumer uses a Tooltip without XAxis and/or YAxis.
+         * In which case, this.state.activeTooltipIndex never gets set
+         * because the mouse events that trigger that value getting set never get trigged without the axis components.
+         *
+         * See FunnelChart story for an example use case
+         */
+        const { data: itemData = [] } = graphicalItem.props;
+        const activeData = itemData.find((datum: { value: number }) => datum.value === activeItem.payload.value);
+        const activeIndex = itemData.indexOf(activeData);
+        return [cloneElement(element, { ...item.props, ...itemEvents, activeTooltipIndex: activeIndex }), null, null];
```

Comment:
I would have rathered set the `activeIndex` prop. However, in the use case of the Funnel `activeIndex` was pre-existing. So setting that to `activeTooltipIndex` would override the prop value if consumer is already passing in `activeIndex` to Funnel.